interface/main/finder/dynamic_finder.php

```diff
@@ -374,11 +374,11 @@ function persistCriteria(el, e) {
             <?php if (AclMain::aclCheckCore('patients', 'demo', '', array('write','addonly'))) { ?>
                 <button id="create_patient_btn1" class="btn btn-primary btn-add" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Add New Patient'); ?></button>
             <?php } ?>
-            <div class="jumbotron mt-3 p-4">
+            <div>
                 <div id="dynamic"><!-- TBD: id seems unused, is this div required? -->
                     <!-- Class "display" is defined in demo_table.css -->
-                    <div class="table-responsive">
```

Comment:
Need to keep table-responsive class so margins and padding are maintained on small screens. Otherwise will scroll on x-axis most times.

Reply:
Added back

Reply:
thanks Robert